### PR TITLE
#164271289 Upgrade Django app from 2.1.2 to 2.1.7

### PR DESCRIPTION
**Pull request by Siffersari**
### What does this PR do?
Upgrades I-reporter Django app version to official release version

### Description of the task to be completed?
- Upgrade I-reporter Django app from Django 2.1.2 to Django 2.1.7

### How should you manually test this?
*Step 1*
 
Navigate to your working directory, create and activate the virtual environment 

```$  virtualenv -p python .venv```

```$ source .venv/bin/activate```

Clone the repository 

```$ git clone https://github.com/ann-mukundi/ireporter.git```

Switch to testing branch

```$ git checkout ch-upgrade-djangoapp-version-164271289 ```

Switch to the ireporter directory

```$ cd ireporter/```

Install project dependencies

```$ pip install -r api/requirements.txt```

*Step 2*

Check Django version

```$ python -m django --version```

### Prerequisites

Python, Git,  Virtualenv

### What are the relevant PT stories?
[#164271289](https://www.pivotaltracker.com/story/show/164271289)

### CODE REVIEW

**Comment by AlvinMugambi** [comment]
Looks good

**Comment by Elisha-Misoi** [comment]
Well done 👍🏽

**Comment by Siffersari** [comment]
Thank you

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+.spyderproject
+.spyproject
+# Rope project settings
+.ropeproject
+# mkdocs documentation
+/site
+# mypy
+*__pycache__
+*/*/__pycache__/
+.vscode/
```
[the remark this answers is not in the document]
Comment (Siffersari) [reply .gitignore]: @katherine95 I have implemented this.

**Pull request merged**